MeetupPlatformApi/DataTransferObjects/MeetupInputDto.cs

```diff
@@ -1,12 +1,20 @@
-﻿namespace MeetupPlatformApi.DataTransferObjects;
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace MeetupPlatformApi.DataTransferObjects;
 
 public class MeetupInputDto
 {
+    [Required]
+    [MaxLength(30)]
     public string Name { get; set; }
 
+    [Required]
     public DateTime StartTime { get; set; }
 
+    [Required]
     public DateTime EndTime { get; set; }
 
+    [Required]
+    [MaxLength(300)]
```

Comment:
Please, extend allowed length ou to something around 4k (almost 50 lines of text with 80 symbols per line).